draft-ietf-quic-transport.md

```diff
@@ -1660,7 +1660,7 @@ Reason Phrase:
 ## MAX_DATA Frame {#frame-max-data}
 
 The MAX_DATA frame (type=0x04) is used in flow control to inform the peer of
-the maximum amount of data that can be sent on the connection as a whole.
+the maximum offset of data that can be sent on the connection as a whole.
```

Comment:
I don't think the concept of an offset for a connection makes sense in QUIC.  It's not TCP, so what we're really talking about is the sum of all consumed stream flow controls.  So I'd suggest: "the maximum consumed stream flow control across all streams so far in the connection." or something similar?